Sources/DistributedActors/Props.swift

```diff
@@ -172,3 +172,17 @@ extension _Props {
         return p
     }
 }
+
+extension _Props {
+    public static func _wellKnownActor(name: String) -> Self {
+        var props = Self._wellKnown
+        props._knownActorName = name
+        return props
+    }
+    
+    public func _knownAs(name: String) -> Self {
+        var p = self
+        p._knownActorName = name
+        return p
+    }
+}
```

Comment:
okey as underscored 👍  the "known as name" stuff we neeed to get rid of but that's on me... okey to use it here.